dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -17,15 +17,30 @@
 #include "utils/datetime.h"
 #include "utils/fileutils.h"
 #include "utils/signal-monitor.h"
+#include "utils/stringutils.h"
+
+static struct rumcontext {
+    char application_id[200];
+    char session_id[200];
+    char view_id[200];
```

Comment:
We use [standard UUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier), so can't we say they are 128-bits (`[16]`)?

Reply:
Wanted to make sure I am safe :)) but you are right I can lower that memory to 32 (just to still be safe but less drastic)